trl/trainer/utils.py

```diff
@@ -226,3 +227,13 @@ def __iter__(self):
                     "input_ids": torch.LongTensor(example),
                     "labels": torch.LongTensor(example),
                 }
+
+
+class PeftSavingCallback(TrainerCallback):
```

Comment:
Should we have an integration test to check this does what we expect (or perhaps you already have it included in the tests?)

Reply:
yes this should be already tested in the sft trainer tests, I just moved it in utils so that the RM Trainer can also benefit from it!